pkg/console/controllers/route/controller_test.go

```diff
@@ -0,0 +1,245 @@
+package route
+
+import (
+
+	// k8s
+
+	"fmt"
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+
+	// openshift
+
+	// console-operator
+
+	"github.com/go-test/deep"
+	routesub "github.com/openshift/console-operator/pkg/console/subresource/route"
+)
+
+const (
+	validCertificate = `
+-----BEGIN CERTIFICATE-----
```

Comment:
thanks, I completely forgot you should also be checking at least the expiry date. For example, this certificate appears to have expired 4 years ago.
```
            Not Before: Apr  5 15:15:55 2013 GMT
            Not After : Apr  4 15:15:55 2015 GM
```